ocs_ci/deployment/acm.py

```diff
@@ -186,7 +186,12 @@ def submariner_configure_upstream(self):
         for i in self.dr_only_list:
             kubeconf_list.append(config.clusters[i].RUN["kubeconfig"])
         connct_check = f"verify {' '.join(kubeconf_list)} --only connectivity"
-        run_subctl_cmd(connct_check)
+        # Workaround for now, ignoring verify faliures
+        # need to be fixed once pod security issue is fixed
+        try:
+            run_subctl_cmd(connct_check)
+        except Exception:
+            logger.warning("Submariner verification has issues")
```

Comment:
We should probably not proceed if submariner has issues.

Reply:
@am-agrawa This was due to a bug in submariner, although deploy was successful, validation was failing. Hence I added this as just warning and proceed.

Reply:
Is that bug resolved? Do we have a bug id?

Reply:
@shylesh ^^

Reply:
@am-agrawa , looks like its fixed , I don't see exception in the latest run https://url.corp.redhat.com/1826235

Reply:
> @am-agrawa , looks like its fixed , I don't see exception in the latest run https://url.corp.redhat.com/1826235

So do we need any code change?

Reply:
may be we could remove the try block